osgi/integrationtest/src/test/java/org/hibernate/validator/osgi/integrationtest/OsgiIntegrationTest.java

```diff
@@ -212,7 +215,7 @@ private ExpressionFactory buildExpressionFactory() {
 		Thread.currentThread().setContextClassLoader( getClass().getClassLoader() );
 
 		try {
-			return ExpressionFactory.newInstance();
+			return ELManager.getExpressionFactory();
```

Comment:
Couldn't we even simplify that entire test by simply relying on the default interpolator?